test/native/tsconfig.json

```diff
@@ -0,0 +1,48 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "target": "esnext",
```

Comment:
kind of copied this from react-native's codebase - still trying to setup a test suite that fails correctly when it should